tasks/test/browser.js

```diff
@@ -37,11 +38,18 @@ module.exports = (gulp) => {
   gulp.task('test:browser', (done) => {
     const runSequence = require('run-sequence')
 
-    runSequence.use(gulp)(
-      'test:karma',
-      'test:karma:webworker',
-      utils.exitOnFail(done)
-    )
+    if (util.env.webworker === 'only') {
```

Comment:
@diasdavid I've made the changes but I can't push to this repo... I can create a new PR or you can give me perms, whatever works.